modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/quote-comparison-header/index.js

```diff
@@ -0,0 +1,21 @@
+function getCookie(name) {
+	name = name + '=';
+	const decodedCookie = decodeURIComponent(document.cookie);
+	const cookies = decodedCookie.split(';');
+	for (let i = 0; i < cookies.length; i++) {
+		const cookie = cookies[i].trim();
+		
+		if (cookie.indexOf(name) == 0) {
+			let cookieObject = cookie.substring(name.length, cookie.length);
+			let firstName = JSON.parse(cookieObject).basics.businessInformation.firstName;
+		
+			return firstName;
+		}
+	}
+}
+
+const firstName = getCookie("raylife-application-form");
```

Comment:
@silasbsi  see this: https://github.com/silasbsi/liferay-portal/commit/a675370cbdf19858a22e857ac1326b493b49d81c